das-dashboard/backend/shared/utils/remote_scp.py

```
@@ -0,0 +1,79 @@
+        ssh = SSHClient()
+        ssh.set_missing_host_key_policy(AutoAddPolicy())
+        ssh.connect(hostname=host, username=username, key_filename=key_path)
+
+        return ssh
+
+    def get_remote_home(self, ssh: SSHClient) -> str:
+        stdin, stdout, stderr = ssh.exec_command("echo $HOME")
+        return stdout.read().decode().strip()
+
+    def remote_file_exists(self, ssh: SSHClient, file_path: str) -> bool:
+        stdin, stdout, stderr = ssh.exec_command(
```

Pylint (4.0.6)

[warning] Unused variable 'stdin'

(W0612)